src/main/java/org/mdbenefits/app/submission/actions/CheckExpeditedSnapEligibility.java

```diff
@@ -3,67 +3,57 @@
-
-            String rentString = inputData.getOrDefault("householdRentAmount", "0").toString();
-            BigDecimal rentAmount = new BigDecimal(rentString).setScale(2, RoundingMode.HALF_UP);
+        if (inputData.getOrDefault("isApplyingForExpeditedSnap", "false").toString().equals("true")) {
 
-            boolean isMigrantOrSeasonalFarmWorker = inputData.get("migrantOrSeasonalFarmWorkerInd").toString().equals("true");
-            // TODO: Update this logic with household expenses that are considered utilities
-            boolean hasUtilities = false;
-//                    inputData.containsKey("householdUtilitiesExpenses[]") && !inputData.get("householdUtilitiesExpenses[]")
-//                            .toString().equals("None");
+            BigDecimal moneyOnHandAmount = convertToBigDecimal(
+                inputData.getOrDefault("expeditedMoneyOnHandAmount", "0").toString());
+            BigDecimal householdIncomeAmount = convertToBigDecimal(
+                inputData.getOrDefault("householdIncomeLast30Days", "0").toString());
 
-            // Household has less than 150 in monthly income, including cash on hand is less than or equal to 100
-            // BigDecimal compareTo returns -1 if less than, 0 if equal, 1 if greater than
-            boolean isEligibleByCashOnHand = householdIncomeAmount.compareTo(INCOME_LIMIT) <= 0
-                    && moneyOnHandAmount.compareTo(MONEY_ON_HAND_LIMIT) <= 0;
-
-            // Is Migrant or Seasonal farmworker household with cash on hand less than or equal to 100
-            boolean isEligibleByMigrantOrSeasonalFarmWorker = isMigrantOrSeasonalFarmWorker
-                    && moneyOnHandAmount.compareTo(MONEY_ON_HAND_LIMIT) <= 0;
-
-            // Households combined monthly income + cash on hand is less than the total monthly rent/mortgage + utilities
-            BigDecimal utilitiesTotal = BigDecimal.ZERO;
-            if (hasUtilities) {
-                // TODO: Update this logic with the relevant household expenses that are considered utilities
-//                List<String> utilityTypes = (List<String>) inputData.get("householdUtilitiesExpenses[]");
-//                utilitiesTotal = utilityTypes.stream()
-//                        .map(type -> inputData.getOrDefault("householdUtilitiesExpenseAmount_wildcard_" + type, "0").toString())
-//                        .map(stringValue -> new BigDecimal(stringValue).setScale(2, RoundingMode.HALF_UP))
-//                        .reduce(BigDecimal.ZERO, BigDecimal::add);
-            }
+            boolean isMigrantOrSeasonalFarmWorker = inputData.getOrDefault("migrantOrSeasonalFarmWorkerInd", "false").toString().equals("true");
+            boolean isBelowMoneyOnHandThreshHold = inputData.getOrDefault("householdMoneyOnHandLessThan100", "false").toString().equals("true");
+            boolean isBelowIncomeThreshHold = inputData.getOrDefault("incomeLessThan150", "false").toString().equals("true");
```

Comment:
(non-blocking nit): Threshold is usually written as one word

```suggestion
            boolean isBelowMoneyOnHandThreshold = inputData.getOrDefault("householdMoneyOnHandLessThan100", "false").toString().equals("true");
            boolean isBelowIncomeThreshold = inputData.getOrDefault("incomeLessThan150", "false").toString().equals("true");
```